cypress/support/commands.js

```diff
@@ -0,0 +1,3 @@
+Cypress.Commands.add('getByDataCy', (selector) => {
+    cy.get(`[data-cy="${selector}"]`);
+  });
```

Comment:
You do not need to write it by yourself. This file should just contain import file from `cypress-tools`

Reply:
ok